bin/oref0-pump-loop.sh

```diff
@@ -709,7 +669,7 @@ function get_settings {
         retry_return read_insulin_sensitivities 2>&3 >&4 || return 1
         retry_return read_carb_ratios 2>&3 >&4 || return 1
         retry_return openaps report invoke settings/insulin_sensitivities.json settings/bg_targets.json 2>&3 >&4 || return 1
-	#NON_X12_ITEMS=""
+    #NON_X12_ITEMS=""
```

Comment:
Tabs vs. spaces? Maybe just remove this old line.

Reply:
This line isn't from me :D I just changed the duty cycle function ;)

Reply:
No, but the diff shows you changed the whitespace such that it is now indented correctly. I’m suggesting either fixing that or removing the line.